### PR TITLE
Docker package and circleci deploy build

### PR DESCRIPTION
**Pull request by dsamojlenko**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,8 +47,30 @@ jobs:
...
+          path: /home/DEV.T1.TBS-SCT.GC.CA/dsamojle-ps/track-web.tar.gz
+          destination: track-web.tar.gz
+          
 workflows:
   version: 2
   tracker:
     jobs:
-      - track_web 
+      - track_web
+      - build_package:
```
Comment (buckley-w-david) [inline .circleci/config.yml]: Can you put a branch filter on this so it only builds master?

File: Dockerfile.build [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:16.04
```
Comment (buckley-w-david) [inline Dockerfile.build]: Do you think this file (or the one below it) need to be present if it's getting built in CI?

I would at least want `build-env.sh` to be put somewhere other than the root of the directory if we're actually adding it to master.

File: .circleci/config.yml (hunk 1 above)
Comment (dsamojlenko) [reply .circleci/config.yml]: ah yes, meant to do that

File: Dockerfile.build (hunk 2 above)
Comment (dsamojlenko) [reply Dockerfile.build]: yeah, I was going to ask your opinion on that - I think we can just remove them?  unless we want to provide an alternate way of generating build packages manually...

**Pull request merged**